CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 CHANGELOG
 =========
 
+## Unreleased
+
+- New example: pulumi-ts [#843](https://github.com/pulumi/pulumi-kubernetes-operator/pull/843)
```

Comment:
I don't think we need a changelog for this. It isn't adding anything to our release bundle, nor does it affect any features.